### DISCUSSION TITLE
Toast onDismiss?

### DISCUSSION THREAD

**Discussion by kidproquo** [chakra-ui/chakra-ui#3168] [Q&A] [answered]
Is there a way to know when a toast is dismissed? I want to clear a field when a toast gets dismissed. I can use the `onClose` callback but that requires the user to explicitly click the close button.

**Comment by chr-ge** [comment, marked as answer]
Try using `onCloseComplete`:
>  Callback function to run side effects after the toast has closed.
   


```jsx
toast({
  title: "Active Toast",
  description: 'This toast is active.',
  status: 'info',
  position: 'bottom',
  duration: 3000,
  isClosable: true,
  onCloseComplete: () => {
    // clear field
  }
})
```
